_posts/2022-11-01-2022-10-monthly-report.md

```diff
@@ -1,3 +1,11 @@
+---
+layout: post
+title: "October 2022 monthly report"
+author: Arthur Cohen

```

Comment:
For all those files the author should be `Philip Herron and Arthur Cohen`

Reply:
Fixed, sorry about that. I used the commit author when filling up this.